raincatcher-generation-utils/README.md

```diff
@@ -0,0 +1,65 @@
+# Raincatcher Generation Utils
+
+This is a utility tool for generating users, workorders and csv data for running
+raincatcher performance tests against. Currently settings are for running tests
+locally.
+
+## Getting Started
+
+Fist install the depenedencies of the tools with:
+
+```
+npm install
+```
+
+### Generating Users
+
+Run the following command to generate the associated user files for the wfm
+demo, whilst changing `numberOfUsers` to the number of users you want to 
+produce.
+
+```
+npm run generate-users -- ./output/ numberOfUsers
+```
+
+This will output two important files in the `passport-users` and `wfm-users`
+folders in the `output` folder. Overwrite
+`raincatcher-core/demo/server/src/modules/passport-auth/users.json` and 
+`raincatcher-core/demo/server/src/modules/wfm-user/users.json` with the produced
+files, and when the server is started the new users will be able to authenticate
+into the system.
+
+A `user_ids.json` an a `users.csv` is also created. The `users.csv` file can be 
+used with JMeter .jmx files for running the performance tests.
+
+Note: Re-running the test will overwrite previously saved files at the
+destination. The default destination is the output folder, in which you can find
+some example data. But you can change the folder by changing `./output/` to the 
+folder you desire. The sub-folders `passport-users` and `wfm-users` must exist in
+the new folder.
+
+
+### Generating Workorders
+
+The mongo database must be running in order for the follow steps to work. The
```

Comment:
This assumes that mongo is localhost. We may have problem with this running on the platform. 

There are two ways to do this:
- use endpoint to generate data
- use platform import in databrowser to append data (may be limited to 40mb json)
Note: you will need to add fh-mbaas-api for databrowser to run.